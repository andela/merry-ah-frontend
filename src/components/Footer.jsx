import React, { Component } from 'react';

export default class Footer extends Component {
  render() {
    return (
      <footer>
        <div>
          <div className="float-left">
            <li><h5>Â© 2019 All Rights Reserved</h5></li>
          </div>
          <div className="float-right">
            <ul>
              <li><a href="#"><i className="fa fa-2x fa-facebook-f"></i></a></li>
              <li><a href="#"><i className="fa fa-2x fa-twitter"></i></a></li>
              <li><a href="#"><i className="fa fa-2x fa-instagram"></i></a></li>
            </ul>
          </div>
        </div>
      </footer>
    )
  }
}
