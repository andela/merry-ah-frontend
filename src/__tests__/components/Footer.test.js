import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer';

let wrapper;
beforeEach(() => {
 wrapper = shallow(<Footer />);
})
describe('Footer Component', () => {
  it('should match snapshot', () => {
      expect(wrapper).toMatchSnapshot();
  })
})
