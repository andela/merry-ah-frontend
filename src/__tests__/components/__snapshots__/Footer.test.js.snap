// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should match snapshot 1`] = `
<footer>
  <div>
    <div
      className="float-left"
    >
      <li>
        <h5>
          Â© 2019 All Rights Reserved
        </h5>
      </li>
    </div>
    <div
      className="float-right"
    >
      <ul>
        <li>
          <a
            href="#"
          >
            <i
              className="fa fa-2x fa-facebook-f"
            />
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            <i
              className="fa fa-2x fa-twitter"
            />
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            <i
              className="fa fa-2x fa-instagram"
            />
          </a>
        </li>
      </ul>
    </div>
  </div>
</footer>
`;
